scripts/build-exe.sh

```
@@ -0,0 +1,118 @@
+# Optional: codesign on macOS, signtool on Windows.
+#
+# Usage:
+#   bash scripts/build-exe.sh
+#
+# CI:
+#   The .github/workflows/release.yml runs this on every tag push and uploads
+#   the artifact to the GitHub Release.
+
+set -euo pipefail
+
+readonly ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)